src/shared/containers/index.tsx

```diff
@@ -73,3 +73,16 @@ export const UsagePage = lazy(() => import('src/usage/UsagePage'))
 export const BillingPage = lazy(() =>
   import('src/billing/components/BillingPage')
 )
+
+export const FunctionsPage = lazy(() =>
```

Comment:
can make this function list page? not usually that person, but we had a problem in notebooks where this pluralized naming convention led to some mishaps during a refactor, just cause it's hard to see the `s`. if no can, no can

Reply:
can can